mne/viz/utils.py

```diff
@@ -1671,48 +1715,45 @@ def on_select(self, verts):
             return
 
         path = Path(verts)
-        inds = np.nonzero([path.contains_point(xy) for xy in self.xys])[0]
-        if self.canvas._key == "control":  # Appending selection.
-            sels = [np.where(self.ch_names == c)[0][0] for c in self.selection]
-            inters = set(inds) - set(sels)
-            inds = list(inters.union(set(sels) - set(inds)))
-
-        self.selection[:] = np.array(self.ch_names)[inds].tolist()
-        self.style_sensors(inds)
+        inds = np.nonzero([path.intersects_path(p) for p in self.paths])[0]
```

Comment:
Just confirming that I understand correctly this change: 

- old behavior used `contains_point` to check if the geometric center of each sensor was within the lasso
- new behavior uses `intersects_path` to compare the lasso to the MPL artists that are rendered for each sensor.

Is that right?  AFAIC either way is OK (but if there was a principled reason *not* to keep doing it the old way, I'm curious to hear what it was).

Reply:
Ah yes, I forgot about this change when I wrote the summary. with `plot_sensors` which draws dots, the change is barely noticable, but `plot_evoked_topo` draws a rectangle. `contains_point` will select the patch if its center is within the lasso, which feels really weird for rectangles.  With this approach, I feel I have very good control over which sensors are selected and which are not.

Reply:
thanks for clarifying. I realized this when I tested it interactively (and noticed the Alt-click problem on Linux), but forgot to come back to remove this question.